src/main/java/pro/sky/java/course2/homework4exceptions/controller/EmployeeController.java

```diff
@@ -20,18 +23,24 @@ public EmployeeController(EmployeeService employeeService) {
 
     @GetMapping(path = "/add")
     public String add(@RequestParam String firstName, @RequestParam String lastName) {
-        employeeService.addEmployee(firstName, lastName);
+        employeeService.addToEmployeeList(firstName,lastName);
```

Comment:
Давайте вернём проделы!
```suggestion
        employeeService.addToEmployeeList(firstName, lastName);
```